docs/src/pages/components/popper/ScrollPlayground.js

```diff
@@ -132,21 +145,35 @@ export default function ScrollPlayground() {
   <Popper
     placement="${placement}"
     disablePortal={${disablePortal}}
-    modifiers={{
-      flip: {
-        enabled: ${flip},
+    modifiers={[
```

Comment:
Modifiers have changed a lot. I didn't want to make any decisions about what to include on this page yet so I included most of the functionality in the flip, preventOverflow and arrow modifiers.